db/schema.rb

```diff
@@ -215,6 +218,18 @@
     t.index ["auth_provider", "uid"], name: "index_providers_on_auth_provider_and_uid", unique: true
   end
 
+  create_table "quotes", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "service_name"
```

Comment:
I think it's a mistake to store the user-facing string value of service names in the database (at the moment if I choose "Plastic Surgeon" in the UI, the DB record will have the value "Plastic Surgeon", not "plastic_surgeon"). The reason being that there some business logic that we will need on subsequent screens where content/fields vary based on the service chosen by the user. We don't want to have to be doing string comparisons against user-facing strings because (a) if ever the wording gets updated we will have to update our business logic and. that will break all the old records and (b) even if we don't think the service needs to work in Welsh, the fact that our business logic would break in Welsh should be a sign we're on the wrong path.

I would strongly suggest that we have two fields here: `service_type` and `custom_service_name`. So then either service_type is something like "plastic_surgeon" and custom_service_name is nil, or service_type is "custom" and custom_service_name is whatever the user-entered string is.

Reply:
yeah I think this is a good suggestion - this extra logic flew past my radar